### ISSUE TITLE
Create Profile Layout (with functionality)

### ISSUE THREAD

**Issue by victor-z-xiong** [JeremyH011/CPEN321#2]
Create front end template for profile screen.

Requirements:
* Picture field
* name field, age field
* personal info field (refer to google docs for fields required) 
  *  https://docs.google.com/document/d/15nnWvrmNtiqpy8W-kXJVff5Aj6hzSFVTp16WvTjTE3g/edit

* Reviews header
  * can just have blank space below for now. Have to figure out ways to add cards to bottom of this page for each review but that will be separate issue. Users should NOT be able to edit their own reviews.

 the tabs should still show up at the bottom (This profile refers to the personal profile of the user that's signed in. (It is editable by the user). Can ask @victor-z-xiong  for clarification

Builds on issue #26 Once users are fully logged in and created an account. They should be able to edit the details (personal information) of their account.

Once issue #16 is done. Can copy that implentation to add photos to profile as well. Can work with @JeremyH011  to finish this.

![image](https://user-images.githubusercontent.com/31559004/64927574-7ac68000-d7c1-11e9-8b07-a8122be57e06.png)
